### ISSUE TITLE
need help

### ISSUE THREAD

**Issue by duoduojia1** [Megum1/LOTUS#6]
Thank you for providing the backdoor mitigation code!  I have reproduced the BadNet baseline using the Backdoor Toolbox on the CIFAR-10 dataset with a ResNet-18 model. However, I found that the finetune method doesn't seem to effectively mitigate BadNet when using an initial learning rate of 2e-2. Could you please share the parameter settings you used for the baselines?😭
